spec/jobs/signal_adapter/attach_contributors_avatar_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe SignalAdapter::AttachContributorsAvatar do
+  describe '#perform_later(contributor)' do
+    subject { -> { described_class.new.perform(contributor) } }
+    let(:contributor) { create(:contributor, signal_phone_number: '+491212343434') }
+
+    context 'with avatar on file system' do
+      let(:avatar) { file_fixture('profile-+491212343434') }
+      before do
+        allow(File).to receive(:file?).with("/app/signal-cli-config/avatars/profile-#{contributor.signal_phone_number}").and_return(true)
```

Comment:
Why are you using string interpolation to construct the path in this line, but not in line 16?